### PR TITLE
New case:RHEL-177342 - Snapshot with guest agent

### PR DESCRIPTION
**Pull request by aliang123**
Signed-off-by: Aihua Liang <aliang@redhat.com>
id:1980231

### CODE REVIEW

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_snapshot_guest_agent.q35: PASS (109.59 s)

**Comment by aliang123** [comment]
@zhencliu @ngu-niny, please help to review, thanks.

File: qemu/tests/blockdev_snapshot_guest_agent.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import logging
+import time
+import random
```
Comment (zhencliu) [inline qemu/tests/blockdev_snapshot_guest_agent.py:3]: we can remove it for random is never used

Comment (aliang123) [reply qemu/tests/blockdev_snapshot_guest_agent.py]: Have updated, thanks zhenchao for pointing it out.

**Comment by zhencliu** [comment]
LGTM

**Comment by ngu-niny** [comment]
(3/6) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_snapshot_guest_agent:  PASS (259.52 s)

Acked-by: Nini Gu <ngu@redhat.com>

**Pull request merged**